<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>951</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>951</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bank Cross-Section Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>750</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="sectionList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addSectionBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeSectionBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="sectionPropTable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addLayerBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeLayerBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="secPropertyViewer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>681</width>
     <height>281</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>增加斷面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeNodesBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>變更</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="noMoveRdo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>No Move</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="retreatRdo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>7</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Retreat</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
